external/mrtrix/mrtrix_track.m

```diff
@@ -1,8 +1,8 @@
-function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose, clobber)
+function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, curvature, cutoff, bkgrnd, verbose, clobber)
 %
 % function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose)
 %
-% Provided a csd estimate, generate estimates of the fibers starting in roi 
+% Provided a csd estimate, generate estimates of the starting in roi 
```

Comment:
let's say "streamlines/fascicles"?

"streamline" is a default term in mrTrix. In that sense, streamline is good.
"fascicle" is widely used for LiFE.